exporters/logging-otlp/src/test/java/io/opentelemetry/exporter/logging/otlp/OtlpJsonLoggingMetricExporterTest.java

```diff
@@ -139,7 +139,7 @@ void log() throws Exception {
             + "  }]"
             + "}",
         logs.getEvents().get(0).getMessage(),
-        /* strict= */ true);
+        /* strict= */ false);
```

Comment:
Not strict allows array comparison without ordering - HashMap happened to (consistently? weird) have the order in the assertion message, while IdentityHashMap changed it due to different internals. But order doesn't matter.